internal/storage/bank_account_repository.go

```diff
@@ -52,7 +52,7 @@ func (p *BankAccountRepository) FindAll(argsStr map[string]string, argsInt map[s
 
 // FindByID ...
 func (p *BankAccountRepository) FindByID(id uint) (model.BankAccount, error) {
-	bankAccount := model.BankAccount{}
+	var bankAccount model.BankAccount
```

Comment:
Same issue.

Reply:
If the record is not exist on db, then response for endpoint `http://localhost:3625/api/bank-accounts/1` will be 
```
{
    "code": 404,
    "status": "Error",
    "message": "record not found",
    "errors": null
}
```
is not ok ?

Reply:
You are right about this response.